### ISSUE TITLE
Search for subtag

### ISSUE THREAD

**Issue by aPaci95** [pneff/anki-hierarchical-tags#7]
I can't understand how I can search for subtag. For example, if I've got the subtag anatomy::stomach, and I want to add a new stomach card, I would need to start typing 'anatomy' for the auto-complete to kick in. Is there a way to just be able to type 'stomach'?

**Comment by glutanimate** [comment]
Hey @aPaci95,

I just wanted to let you know that I've just completed [an add-on](https://github.com/glutanimate/anki-addons-misc/blob/master/src/tagedit_subtag_completer.py) that should cover your use case. By default it will match arbitrary tag substrings, but you can also configure it to only match hierarchical (sub)tags (see the config section at the top for more information).

The story behind this is actually quite interesting. Another user found [your Reddit thread](https://www.reddit.com/r/Anki/comments/69mbp0/addon_request_about_tags/) while looking for ways to improve Anki's tag system and ended up commissioning me to write this add-on. So most credit for this goes to them!

In any case, I hope you'll find the add-on useful.

Best,
Glutanimate

PS: If you're still interested in the other features you listed, I'd be more than happy to write an add-on for them as well. The click-and-drag option might be very tricky to implement, or maybe not possible at all, but the other two should be feasible. Just drop me an email at ankiglutanimate [αt] gmail . com if you're interested and we can discuss the details of the commission. (Sorry if this is the wrong place to post this)

**Comment by ELLIOTTCABLE** [comment]
Thank you, @glutanimate! That link is broken, though.

I found this 11 days after you posted it, after coming here to copy the following request from an Ankiweb commenter:

> For example, if I've got the subtag language::japanese, and I want to add a new japanese card, I would need to start typing 'language' for the auto-complete to kick in, while it would be super handy for me to just be able to type 'japanese'. Anyways, it's just a minor nitpick in an otherwise great addon!

Meanwhile, I'd still love to see this built-in to AHT. <3

**Comment by glutanimate** [comment]
Yeah, I recently restructured my add-on repo for 2.1. [Here is an updated link](https://github.com/glutanimate/anki-addons-misc/blob/master/src/tagedit_subtag_completer/tagedit_subtag_completer.py).

I don't know how @pneff feels about this, but I'd be happy to submit a PR that integrates this change into Hierarchical Tags.

**Comment by ELLIOTTCABLE** [comment]
@glutanimate I'm rather new to Anki, but an old-school programmer — if it's not on Ankiweb, as yours isn't, is there any particular thing I need to do other than adding the file by its lonesome to the add-ons folder?
